### ISSUE TITLE
Updating fields vanishes the other fields.

### ISSUE THREAD

**Issue by beratuslu** [thisbejim/Pyrebase#234]
Hello,
I am trying to update a child's only one field like following, when I run the below code it vanishes, "anotherField"s do you guys any idea how to to it without touching "anotherField"s?

Thank you..

My firebase db is like:
`{
  "Los Angeles" : {
    "anotherField" : "something",
    "isSunny" : false
  },
  "istanbul" : {
    "anotherField" : "something",
    "isSunny" : false
  }
}
`

I am trying to do update with the following data:
`dict = {
  "Los Angeles" : {
    "isSunny" : true
  },
  "istanbul" : {
    "isSunny" : true
  }
}
db.child("weather").update(dict)
`

**Comment by mikebski** [comment]
You need to update further down the tree, so you would do 2 updates, one to "Los Angeles" with {"isSunny": true} and one to "istanbul" with {"isSunny": true}

This is not a bug

**Comment by beratuslu** [comment]
Yes, I meaned is there a way to do this with one single reequest?

**Comment by beratuslu** [comment]
https://stackoverflow.com/questions/47814463/pyrebase-updating-fields-vanishes-the-other-fields/47815866#47815866
